ci-operator/step-registry/ipi/conf/aws/oidc-creds-provision/ipi-conf-aws-oidc-creds-provision-commands.sh

```diff
@@ -7,7 +7,14 @@ set -o pipefail
 MPREFIX="${SHARED_DIR}/manifest"
 TPREFIX="${SHARED_DIR}/tls"
 infra_name=${NAMESPACE}-${UNIQUE_HASH}
-export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+
+if [[ -f "${SHARED_DIR}/aws_minimal_permission_ccoctl" ]]; then
```

Comment:
How about using `AWS_CCOCTL_USE_MINIMAL_PERMISSIONS` to decide to use which credential file, so that, if `ipi-conf-aws-oidc-creds-provision` is referenced in a wrong place in a install workflow, we can detect the failure easily.

Reply:
Updated.